publication/src/main/scala/hmda/publication/reports/disclosure/DisclosureReportPublisher.scala

```diff
@@ -77,6 +75,8 @@ class DisclosureReportPublisher(supervisor: ActorRef) extends HmdaActor with Loa
     D71, D72, D73, D74, D75, D76, D77,
     D81, D82, D83, D84, D85, D86, D87,
     D11_1, D11_2, D11_3, D11_4, D11_5, D11_6, D11_7, D11_8, D11_9, D11_10,
+    A1, A2, A3,
+    A1W, A2W, A3W,
```

Comment:
I believe the reports with `W` in the name are all supposed to be the nationwide totals, not limited to specific MSAs. I looks like we'll need to generate them separately from the whole set of LARs, instead of the LAR set filtered by MSA.